CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+# Contributing to the Project
+
+This document contains and defines the rules that have to be followed by any
+contributor to the project, in order for any change to be merged into the
+stable branches.
+
+## Workflow
+
+The workflow followed is waterfall. In short, a contributor will target his 
+work on the lowest version branch possible starting with 2.0, and the changes
+will automatically be forward-ported to higher versions.
+
+## Restrictions of the master branch
```

Comment:
There is no `master` branch, so this is a bit confusing.

Reply:
Changed to `development/X.Y`